project/backend/api-gateway/docker-compose.yml

```
@@ -0,0 +1,21 @@
+      # Point to the SHARED collector using its container name
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4318
+      - OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+      - OTEL_SERVICE_NAME=api-gateway
+      # Add any other specific environment variables for the API gateway
+    networks:
+      - monitoring_net # Connect to the shared network
+      - default # Keep default network
+
+networks:
+  monitoring_net:
+    external: true # Specify that this network is created elsewhere
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)